core/node/api_server/src/execution_sandbox/storage.rs

```diff
@@ -56,6 +56,10 @@ impl<S: ReadStorage> StorageWithOverrides<S> {
                 let code_key = get_code_key(account);
                 let code_hash = code.hash();
                 self.overridden_slots.insert(code_key, code_hash);
+                // FIXME: is this addition correct?
+                let known_code_key = get_known_code_key(&code_hash);
+                self.overridden_slots
+                    .insert(known_code_key, H256::from_low_u64_be(1));
```

Comment:
Without adding a bytecode to the known bytecodes storage, decommitting it via the code oracle will fail (which seems to differ from how real contracts work). OTOH, I'm not 100% certain that adding this slot override won't have negative side effects.

Reply:
Yep, that's correct

Reply:
Sorry, I phrased it really weirdly. I meant that if this additional log is not added, decommitments don't work; deploying real contract adds such a log, so decommitment works for them.